CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.4.0] - 2016-08-04
```

Comment:
You'll also need to add a section at the bottom of the file so that the `[0.4.0]` links to the compare view for the change. You can see an example of doing that in https://github.com/everypolitician/scraped_page_archive/commit/de88f2ffc7acc6ddc31c3818c5465e44d7d10943#diff-4ac32a78649ca5bdd8e0ba38b7006a1e.